lib/fast_jsonapi/object_serializer.rb

```diff
@@ -111,11 +111,11 @@ def set_key_transform(transform_name)
           dash: :dasherize,
           underscore: :underscore
         }
-        @transform_method = mapping[transform_name.to_sym]
+        self.transform_method = mapping[transform_name.to_sym]
```

Comment:
I found it is odd that every other class instance variable is accessed via accessor and only this one is not. So I changed it to accessor!